internal/core/domain/link.go

```diff
@@ -74,6 +75,7 @@ type Link struct {
 	Active                   bool
 	Schema                   *Schema
 	IssuedClaims             int // TODO: Give a value when link redemption is implemented
+	RefreshService           *verifiable.RefreshService
```

Comment:
missing this field in `domain.Claim` struct as well. This field should be populated at least in the repository function `func (c *claims) GetByIdAndIssuer` so mobile can fetch this credential with the proper RefreshService field

Reply:
+1